docs/tests/integration/test_agentd/test_agentd_reconnection.md

```diff
@@ -1,3 +1,44 @@
+# Test AgentD reconnection
+
+## Overview
+
+These tests will check that, during enrollment, the agent re-establishes communication with the manager
+under different situations that interrupt it.
+
+## Objective
+
+The objective is to check that, with different states in the clients.key file, the agent
+successfully enrolls after losing connection with remoted.
+
+## General info
+
+|Tier | Platforms | Number of tests | Time spent |
+|:--:|:--:|:--:|:--:|
+| 0 | Linux/Windows | 10 | 17m 53s |
+
+## Expected behavior
+
+Success if the agent enrolls and starts communication with remoted, failure otherwise.
+
+## Testing
+
+The tests are based on verifying that AgentD is communicating with RemoteD (if the agent is already enrolled),
```

Comment:
```suggestion
The tests are based on verifying that wazuh-agentd is communicating with wazuh-remoted (if the agent is already enrolled),
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca